src/sickle.ts

```diff
@@ -14,10 +14,12 @@ export function formatDiagnostics(diags: ts.Diagnostic[]): string {
       .join('\n');
 }
 
-export type AnnotatedProgram = {
+export type StringMap = {
```

Comment:
`error TS4081: Exported type alias 'AnnotatedProgram' has or is using private name 'StringMap'.`

Which makes sense, I guess?